content-api/content-actors/src/main/scala/org/sunbird/content/actors/ContentActor.scala

```diff
@@ -144,6 +144,7 @@ class ContentActor @Inject() (implicit oec: OntologyEngineContext, ss: StorageSe
 	def uploadPreSignedUrl(request: Request): Future[Response] = {
 		val `type`: String = request.get("type").asInstanceOf[String].toLowerCase()
 		val fileName: String = request.get("fileName").asInstanceOf[String]
+		val mimeType: String = request.get("mimeType").asInstanceOf[String]
```

Comment:
Why are we looking for mimeType here?

Reply:
Only for GCP we are sending. For AWS & Azure even though request doesn't have, it's not breaking.